docs/concepts/jcloud/yaml-spec.md

```diff
@@ -161,10 +161,11 @@ If your Executor needs high IO, you can use `ebs` instead. Please note that:
 - You must pass a storage size parameter (default: `1G`, max `10G`).
 
 ````
+JCloud also supports retaining the data a Flow was using while active. You can set the `retain` argument to `true` to enable this feature.
```

Comment:
Are we calling it JCloud or JIna AI Cloud?

Reply:
The [docs](https://docs.jina.ai/concepts/jcloud/yaml-spec/) use `JCloud` so I followed that here as well